profit/run/default.py

```diff
@@ -221,13 +221,86 @@ class TemplatePreprocessor(Preprocessor):
     """
     def pre(self, data, run_dir):
         # No call to super()! replaces the default preprocessing
-        from profit.pre import fill_run_dir_single
         if os.path.exists(run_dir):
             rmtree(run_dir)
-        fill_run_dir_single(data, self.config['path'], run_dir, ignore_path_exists=True,
+        self.fill_run_dir_single(data, self.config['path'], run_dir, ignore_path_exists=True,
                             param_files=self.config['param_files'])
         os.chdir(run_dir)
 
+    def fill_run_dir_single(self, params, template_dir, run_dir_single, param_files=None, overwrite=False,
```

Comment:
All the methods which were moved from `pre` should be `@staticmethod` as they don't require/modify the Preprocessor object.

Alternatively it might even be better to move the whole Template-Preprocessor and it's associated functions (and maybe even the Preprocessor ABC) into their own file.